k4EDM4hep2LcioConv/src/k4EDM4hep2LcioConv.cpp

```diff
@@ -661,7 +667,12 @@ lcio::LCCollectionVec* convMCParticles(
     if (edm_mcp.isAvailable()) {
       lcio_mcp->setPDG(edm_mcp.getPDG());
       lcio_mcp->setGeneratorStatus(edm_mcp.getGeneratorStatus());
-      // lcio_mcp->setSimulatorStatus(edm_mcp.getSimulatorStatus());
+      int status = edm_mcp.getGeneratorStatus();
+      lcio_mcp->setSimulatorStatus(status);
```

Comment:
Maybe also add a comment here, that this works as expected as long as things stay in memory, but seem to break when they hit the disk. We have yet to understand if this is a "real" problem, or if we are just missing something here.

Reply:
If this is kept, then also add it to the release notes.
But is this cannot be persisted, then we probably shouldn't do that. 
This was not done for some reason previously (https://github.com/key4hep/k4MarlinWrapper/commit/0b5aab5c582e53f0eb6720332675198eebf99dcc), but I have no memory or record of why

Reply:
What we found in the ongoing work for a standalone converter is that some of the bits seem to be set during I/O, which makes a direct comparison of the resulting integer value from `getSimulatorStatus` impossible. We haven't check whether the interpretation of the "important" bits are affected by this.

Reply:
Yes, things run, but a "trivial" comparison fails

Reply:
@FinnJohannsen Just add a comment for this line, that the values get modified at runtime for some reason as Thomas wrote